src/lib/ursadb.py

```diff
@@ -43,16 +42,17 @@ def make_socket(self, recv_timeout: int = 2000) -> zmq.Context:
     def query(
         self,
         query: str,
-        taint: Optional[str] = None,
+        taint: List[str] = None,
         dataset: Optional[str] = None,
     ) -> Json:
         socket = self.make_socket(recv_timeout=-1)
 
         start = time.clock()
         command = "select "
         if taint:
-            taint = taint.replace('"', '\\"')
-            command += f'with taints ["{taint}"] '
+            taint_str = str(taint)
+            taint_str = taint_str.replace("'", '"')
```

Comment:
what if taint name contains `"` or `'`?

You should create them one by one and then `", ".join` instead of relying on `str()`